packages/powersync/lib/src/database/powersync_db_mixin.dart

```diff
@@ -239,6 +239,8 @@ mixin PowerSyncDatabaseMixin implements SqliteConnection {
         await tx.execute('DELETE FROM ${quoteIdentifier(row['name'])}');
       }
     });
+    // The data has been deleted - reset these
+    setStatus(SyncStatus(lastSyncedAt: null, hasSynced: false));
```

Comment:
Do we have this same issue on the JS SDK? I see we do not reset the status in `disconnectAndClear()`.

Reply:
We might have the same issue there also. I haven't tested it yet, but from looking at the code quickly it seems like it should have the same issue.